vmdb/app/models/server_role.rb

```diff
@@ -52,23 +52,23 @@ def self.to_role(server_role)
   end
 
   def self.all_names
-    self.find(:all, :order => :name, :select => "id, name").collect { |r| r.name }
+    self.order(:name).pluck(:name)
   end
 
   def self.database_scoped_role_names
-    self.find_all_by_role_scope('database', :select => "id, name", :order => :name).collect { |r| r.name }
+    self.where(:role_scope => 'database').order(:name).pluck(:name)
   end
 
   def self.database_scoped_roles
-    @database_scoped_roles ||= self.find_all_by_role_scope('database').sort { |a,b| a.name <=> b.name }
+    @database_scoped_roles ||= self.where(:role_scope => ('database').order(:name)).to_a
```

Comment:
I think the parens are wrong here.  The parens on database are useless and when you remove them you see .order being called on a String.